README.md

```diff
@@ -30,9 +30,9 @@ Finding device ids is tricky, so I recommend using the dropdown in the visual ca
 This card locates your OpenSprinkler entities by using their entity ids. If you haven't changed these, you have nothing to worry about.
 
 Otherwise, make sure:
-- The ids of Station status sensors end with `_status`
-- The ids of Program running binary sensors end with `_program_running`
-- The id of the Opensprinkler Enable switch ends with `opensprinkler_enabled`
```

Comment:
This was the main thing I wanted to fix, the `opensprinkler` part depends on what you name the device when you add the integration (opensprinkler is default) so if you have multiple devices this will fail for the second one or if you name the device anything but the default